README.md

```diff
@@ -62,12 +60,6 @@ In both modes, system-level metrics (e.g. tracepoints), are always grouped by th
 
    `sudo sysctl kernel.perf_event_paranoid=1`
 
- * Tracepoints, block I/O, syscalls and system-wide monitoring on kernels older than 4.3 requires access to debugfs.
```

Comment:
We still need access to debugfs, just not because of system-monitoring.

Reply:
This will be addressed in the "improve tracepoint documentation" PR